src/Symfony/Component/Console/Application.php

```diff
@@ -335,6 +335,8 @@ public function register($name)
     /**
      * Adds an array of command objects.
      *
+     * @see Application::add()
```

Comment:
like `{@inheritdoc}` ? not sure, last time I played around with Sami it would not result in the docs. you prop. want, but happy to put it in there